x/mongo/driver/topology/connection.go

```diff
@@ -46,6 +46,8 @@ type connection struct {
 	connectDone      chan struct{}
 	connectErr       error
 	config           *connectionConfig
+	cancelConnect    context.CancelFunc
```

Comment:
Can these both be renamed to something more descriptive? e.g. `cancelConnectContext` and `connectContextMade`?